remotespark/utils/sparkevents.py

```diff
@@ -76,20 +77,65 @@ def emit_statement_execution_end_event(self, session_guid, language, session_id,
                        (constants.LIVY_KIND, language),
                        (constants.SESSION_ID, session_id),
                        (constants.STATEMENT_GUID, statement_guid),
-                       (constants.STATEMENT_ID, statement_id)]
+                       (constants.STATEMENT_ID, statement_id),
+                       (constants.SUCCESS, success),
+                       (constants.EXCEPTION_TYPE, exception_type),
+                       (constants.EXCEPTION_MESSAGE, exception_message)]
 
         self._send_to_handler(kwargs_list)
 
-    def emit_magic_execution_start_event(self, magic_name, language, magic_guid):
-        self._emit_magic_execution_event(constants.MAGIC_EXECUTION_START_EVENT, magic_name, language, magic_guid)
+    def emit_sql_execution_start_event(self, session_guid, language, session_id, sql_guid):
+        self._verify_language_ok(language)
+
+        event_name = constants.STATEMENT_EXECUTION_START_EVENT
```

Comment:
Do we have a difference between Statement Execution & Sql excecution, or they are the same?